README.md

```diff
@@ -1,6 +1,6 @@
 # Steamship Python Client Library For LangChain (🦜️🔗)
 
-[![stability-alpha](https://img.shields.io/badge/stability-alpha-f4d03f.svg)](https://github.com/mkenney/software-guides/blob/master/STABILITY-BADGES.md#alpha)
+[![Steamship](https://raw.githubusercontent.com/steamship-core/python-client/main/badge.svg)](https://www.steamship.com/build/langchain-apps?utm_source=github&utm_medium=badge&utm_campaign=github_repo&utm_id=github_langchain_repo)
```

Comment:
i'm confused by this appearing in the diff, as this doesn't appear on main any more. Any idea what is happening here?